rpms/python-celery/python-celery.spec

```diff
@@ -1,26 +1,26 @@
-Name:           python-celery
-Version:        3.1.11
-Release:        2%{?dist}
-BuildArch:      noarch
-
-License:        BSD
-Group:          Development/Languages
-Summary:        Distributed Task Queue
-URL:            http://celeryproject.org
-Source0:        http://pypi.python.org/packages/source/c/celery/celery-%{version}.tar.gz
-
-BuildRequires:  python2-devel
-BuildRequires:  python-setuptools
-
-Requires:       python-anyjson
-Requires:       python-dateutil
-Requires:       python-kombu >= 3.0.15
-Requires:       python-setuptools
-Requires:       pyparsing
-Requires:       python-billiard >= 3.3.0.17
-Requires:       python-amqp
-Requires:       pytz
-
+%{?scl:%scl_package python-celery}
+%{!?scl:%global pkg_name %{name}}
+
+%global srcname celery
+
+Name:		%{?scl_prefix}python-celery
+Version:	4.0.2
+Release:	2%{?dist}
+Summary:	Distributed Task Queue
+
+License:	BSD
+URL:		http://celeryproject.org
+Source0:	https://github.com/celery/celery/archive/v%{version}.tar.gz
+
+BuildRequires:  %{?scl_prefix_python}python-devel
+BuildRequires:  %{?scl_prefix_python}python-setuptools
+BuildRequires:  %{?scl_prefix}python-billiard
+BuildRequires:  %{?scl_prefix}python-kombu
+Requires:  %{?scl_prefix}python-amqp
```

Comment:
Remove this since technically celery does not require amqp, it requires kombu which requires amqp.